lib/blockchain/chain.js

```diff
@@ -1605,14 +1605,15 @@ class Chain extends AsyncEmitter {
     this.emit('tip', entry);
     this.emit('reconnect', entry, block);
 
+    if ((entry.height % this.network.names.treeInterval) === 0)
+      this.emit('tree commit', this.db.tree.rootHash(), block, entry);
```

Comment:
I notice that the `entry` seems to come before the `block` in the other events. Going to update these changes so that it is more consistent with the other events.